onmt/Loss.py

```diff
@@ -27,7 +27,7 @@ def nmt_criterion(vocab_size, gpuid, pad_id):
     return crit
 
 
-class Statistics:
+class Statistics(object):
```

Comment:
Maybe move this to a new file called Log.py?  Possibly removing the score() method and just making a local function.